fbcnms-packages/fbcnms-platform-server/features.js

```diff
@@ -245,6 +245,12 @@ export const arrayConfigs = [
     title: 'Work Order New Status',
     enabledByDefault: false,
   },
+  {
+    id: 'projects_bulk_upload',
+    title: 'Project Bulk Upload',
+    enabledByDefault: false,
```

Comment:
this will make sure we have turned on by default in localhost and test environments
```suggestion
    enabledByDefault: false,
    rules: [AlwaysEnabledInTestEnvRule],
```